docs/getting_started/install_manually.md

```diff
@@ -335,160 +337,142 @@ See [Set up directory permissions](set_up_directory_permissions.md) for more inf
 
-a. Download archive from [ezplatform.com](https://ezplatform.com/#download-option). Extract the eZ Platform archive to a directory, then execute post install scripts.
+Create new database. Run following command inside mysql shell:
 
 ``` bash
-cd /<directory>/
-php -d memory_limit=-1 composer.phar run-script post-install-cmd
+CREATE DATABASE ezplatform;
 ```
 
-b. Copy the virtual host template:
+#### Install eZ Platform
+
+Create `clean` installation in development environment with:
 
 ``` bash
-COPY c:\Program Files\Apache Software Foundation\Apache2.2\vhost.template c:\Program Files\Apache Software Foundation\Apache2.2\users/ez1.lh.conf
+composer ezplatform-install
 ```
 
-d. Modify virtual host file **vhost.template.**
+If you had not created database earlier this command may do so. Before executing it make sure that the user set during the Composer install has sufficient permissions. 
 
-Replace the `---USER_ID---` variable (used in lines 10 and 17) with your current user ID. Use `whoami` command to get effective user ID of the currently logged user. If you want to use the default virtual host template (delivered with eZ Platform package) all you have to do is set up lines 7, 8, 9, 10, 17, 25 and 33:
+If Composer asks for your token, you must log in to your GitHub account and generate a new token
+(edit your profile, go to Developer settings > Personal access tokens and Generate new token with default settings).
+This operation is performed only once when you install eZ Platform Enterprise Edition for the first time.
 
-e. Restart Apache 2 server:
+!!! enterprise
 
-``` bash
-httpd.exe -k restart
-```
+    ###### Enable Date-based Publisher
 
-f. Install required dependencies using Composer:
+    To enable delayed publishing of Content using the Date-based Publisher, see [above](#enable-date-based-publisher)
 
-``` bash
-composer install
-```
+!!! tip
 
-When Composer asks you for the token you must log in to your GitHub account and edit your profile. Go to the Personal access tokens link and Generate new token with default settings. Be aware that the token will be shown only once, so do not refresh the page until you paste the token into Composer prompt. This operation is performed only once when you install eZ Platform for the first time.
+    You can use PHP's built-in server after installation: `php bin/console server:start`.
 
-h. Install eZ Platform:
+    If you want to use an Apache web server, you need to prepare a Virtual Host and [set up directory permissions](set_up_directory_permissions.md)
 
-``` bash
-php bin/console ezplatform:install ezplatform-clean
-```
 
-You will be able to see your page under <http://ez1.lh> (or the address you chose in preparation). Please note that a clean install of eZ Platform doesn’t include DemoBundle anymore.
+### Set up Virtual Host
 
-!!! enterprise
+1. To set up Virtual Host use the template provided with eZ Platform. You can find it in `ezplatform\doc\apache2` directory.
```

Comment:
There are two Apache servers available for Windows: 2.2 and 2.4.
Maybe we should mention that template for 2.4 is `vhost.template` and for 2.2 `vhost.2.2.template` should be used.

Reply:
Added to the next point, as we mention in it `Copy the Virtual Host template adequate to your Apache version`